_posts/0000-01-02-anakellyc.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
"Every **great** developer you know got there by solving problems they were *unqualified* to solve until they actually did it." - Patrick McKenzie